### ISSUE TITLE
Incorrect line number in XML export when previously scanned CFML-based file

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#208]
The XML export contains incorrect line numbers for files, when a preceding file scanned in a folder scan was CFML-based with unclosed tags.

## Steps to reproduce

1. Create a folder, say `model`, and inside it two files (note the names, they need to be alphabetically ordered this way):

    `file1.cfc`

    ```
    <cfcomponent>
        <cfproperty name="one" type="string">
        <cfproperty name="two" type="string">
        <cfproperty name="three" type="string">
        <cfproperty name="four" type="string">
        <cfproperty name="five" type="string">
    </cfcomponent>
    ```

    `file2.cfc`

    ```
    component {
        public string function fooFunction() {
            return foo = bar;
        }
    }
    ```
1. Run `java -jar CFLint-0.8.0-all.jar -folder models -xml`
1. The output file `cflint-result.xml` will contain:

    ```xml
    <issue severity="ERROR" id="MISSING_SEMI" message="MISSING_SEMI" category="CFLint" abbrev="MS">
      <location file="C:\Inbox\cflint-bug\models\file2.cfc" fileName="file2.cfc" function="" column="19" line="8" message="End of statement(;) expected instead of =" variable="">
          <Expression><![CDATA[=]]></Expression>
      </location>
    </issue>
    ```

    While `file2.cfc` only contains 5 lines, and the actual issue reported is in line 3. It added the number of `cfproperty` lines in `file1.cfc` to the line number, producing `5 + 3 = 8`.
